config/initializers/sufia.rb

```diff
@@ -144,6 +144,7 @@
 
   # Specify the path to the file characterization tool:
   # config.fits_path = "fits.sh"
+   config.fits_path = "fits"
   config.fits_path = "fits"
```

Comment:
@criedlberger Committing configuration changes here - I'm not sure whether we still need to keep this path as something that gets set on installation, but as long as it is we shouldn't add multiples.